aerial_autonomy_stacks.md

```diff
@@ -75,3 +75,19 @@ Working list:
 A list of packages which don't comprise a full stack but do offer value on top of basic flight controller firmware.
 
 * [MAVROS Controllers](https://github.com/Jaeyoung-Lim/mavros_controllers)
+
+## Indoor navigation packages
+ROS compatible indoor navigation packages that were developed and tested on aerial robotic platforms are listed below. The list, though not exhaustive, provides a good overview of off-the-shelf indoor navigation software. 
+
+| Package name        | OS/OC | Sensors required         | ROS support | Simulator | Platform/controller | Last updated  |
+| ------------------- | ----- | ------------------------ | ----------- | ----------| ------------------- | ------------- |
+| [Cartographer ROS](https://github.com/ArduPilot/ardupilot_ros/tree/humble)   | ✓     | LiDAR        | ROS 2 | Gazebo | Iris coptor,Ardupilot | 02/2024 |
```

Comment:
```suggestion
| [Cartographer ROS](https://google-cartographer-ros.readthedocs.io/en/latest/)   | ✓     | LiDAR        | ROS 2 | Gazebo | Iris coptor,Ardupilot | 02/2024 |
```

Reply:
The Ardupilot link shows a repo that implements Cartographer on an aerial vehicle in ROS.  The Cartographer link is more generic, leaving the user to figure out the autopilot and aerial vehicle specifics. The former is probably more useful for getting started quickly, which is why it was included in the list.

Reply:
ahh oke, I thought that the link was perhaps a copy paste mistake. No worries then!